.github/workflows/backend_ci.yml

```
@@ -0,0 +1,157 @@
+                    echo "test_status=success" >> $GITHUB_OUTPUT
+                  fi
+              env:
+                  SPRING_DATASOURCE_URL: jdbc:postgresql://localhost:5432/airing
+                  SPRING_DATASOURCE_USERNAME: postgres
+                  SPRING_DATASOURCE_PASSWORD: postgres
+                  GRADLE_OPTS: -Dorg.gradle.daemon=false
+                  JAVA_OPTS: -Xmx2048m
+
+            - name: Comment Test Results
+              if: always()
+              uses: actions/github-script@v6
```

actionlint (1.7.7)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)